src/clojure/nrepl/server.clj

```diff
@@ -64,6 +64,31 @@
                           s))
                       % %))))
 
+(defn no-auth-wrapper
```

Comment:
I'm not sure I understand the purpose of this wrapper. I guess that if we don't want to use authorization we can simply pass `nil` or something along those lines.

Reply:
Assuming I understand, that'd be fine too.  Were you saying that you just wanted to special case the "no auth" case at the point of use, i.e. put this code there guarded by a check whether the default-auth-wrapper is set, or are you saying that you don't want to warn at all when there's no protection?